tests/test_definitions/column_aggregate_expectations/expect_column_max_to_be_between.json

```diff
@@ -87,7 +117,34 @@
         "success": false,
         "observed_value": 5
       }
-    }]},
+    },{
```

Comment:
❤️ @jcampbell Thanks for updating the test.  I would like to cherry pick it into my pull request (and will let you know).  Thank you.